gnat2goto/driver/tree_walk.adb

```diff
@@ -1733,6 +1737,23 @@ package body Tree_Walk is
       return Ret;
    end Do_Constant;
 
+   ---------------------------
+   -- Do_Character_Constant --
+   ---------------------------
+
+   function Do_Character_Constant (N : Node_Id) return Irep is
+      Ret : constant Irep := New_Irep (I_Constant_Expr);
+      Resolved_Type : constant Irep := Do_Type_Reference (Etype (N));
+      Constant_Width : constant Integer := 8;
```

Comment:
I do not believe characters are guaranteed to be 8 bit; For example there are `Wide` and `Wide_Wide` characters

Reply:
Basically if you're saying we won't support these for now I think that's fine but this should both be documented and made explicit in the code.

Reply:
Yes this was an assumption I wanted someone else to check. I will see how to handle this.